src/successModal/SuccessModal.tsx

```diff
@@ -58,15 +58,16 @@ class SuccessModal extends Component<SuccessModalProps> {
         forwardedRef={forwardedRef}
         className={className}
         modalContentClassName={baseClassName}
-        ariaLabelledBy={ariaLabelledBy}
-        ariaDescribedBy={ariaDescribedBy}
+        ariaLabelledBy={successContentId}
       >
         <img
           className={`${baseClassName}-bodyItem ${baseClassName}-image`}
           src={imageSrc}
           alt={imageText}
+          // This image is always decorative
+          aria-hidden={true}
```

Comment:
Shouldn't we just put an empty `alt` then?

Reply:
That's what we do with the default value. I let the possibility to add a description (SEO purpose, image not loaded...).